src/models/player.rs

```diff
@@ -860,7 +862,8 @@ fn subtitles_update<E: Env + 'static>(
                         .extend_one(
                             &VIDEO_SIZE_EXTRA_PROP,
                             video_params.size.as_ref().map(|size| size.to_string()),
-                        ),
+                        )
+                        .extend_one(&VIDEO_FILENAME_EXTRA_PROP, video_params.filename.to_owned()),
```

Comment:
Since this is going to be part of `Stream.behaviourHints` object, maybe we should take from there?

Reply:
Tho that means that every client will have to duplicate this logic, extracting values from `Stream.behaviourHints` and passing them via `VideoParams` instead of core taking care of that.